tests/ImageSharp.Tests/Memory/Allocators/ArrayPoolMemoryAllocatorTests.cs

```diff
@@ -186,7 +186,15 @@ static void RunTest()
             {
                 StaticFixture.MemoryAllocator = ArrayPoolMemoryAllocator.CreateDefault();
 
-                Assert.False(StaticFixture.CheckIsRentingPooledBuffer<SmallStruct>(2 * 4096 * 4096));
+                if (TestEnvironment.IsWindows)
+                {
+                    // TODO: We should have an attribute for this kind of stuff.
+                    // This test passes locally but not in the UNIX CI.
+                    // This could be due to the GC simply returning the same buffer
+                    // from unmanaged memory but this requires confirmation.
+                    Assert.False(StaticFixture.CheckIsRentingPooledBuffer<SmallStruct>(2 * 4096 * 4096));
+                }
```

Comment:
Can be easily done via https://github.com/SixLabors/ImageSharp/pull/1657
But not sure if `RuntimeFeatures.Windows` would not look ugly alongside things like `Avx` though

Reply:
Arcade has tons of stuff like that actually we could steal.
https://github.com/dotnet/arcade/blob/12956a100f3d7897456f8d1b8572b2329478aa52/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/PlatformSpecificAttribute.cs

Reply:
Oh that's like the exact thing you wanted!
Strange that they didn't implement something like that for intrinsics.